client/pages/Login/index.css

```diff
@@ -1,10 +1,23 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,900&display=swap');
 
 
+
+@media all and (max-width: 1200px) {
+  body {
+  font-family: 'Roboto', sans-serif;
+  background-repeat: no-repeat;
+  box-shadow:inset 0 0 0 2000px rgba(6, 40, 63, 0.444);
+  background-image: url('../../assets/image/pexels-photo-2422290.png') !important;
+  background-color: rgba(70, 129, 95, 0.444) !important;
+  background-size: auto !important;
+}
+}
+
```

Comment:
I think the media query is not really necessary. The previous style applies to scene sizes